### PR TITLE
Add section about the tutorial updates to the release docs

### PR DESCRIPTION
**Pull request by mansona**

### CODE REVIEW

**Review by jaredgalanis — APPROVED** [review]

File: ember-releases.md [hunk 1]
```diff
@@ -6,9 +6,18 @@ the learning team has to make sure some parts of our infrastructure are adequate
...
+1. [Create the release blog post](#create-the-release-blog-post)
 1. [Deprecations](#deprecations)
 
+### Merge Tutorial Updates
+
+💁 manual process
+
+🚨 This process is not well defined and needs to be streamlined ASAP 🚨
+
+Check if there are any [`Tutorial Updates` PRs](https://github.com/ember-learn/guides-source/pulls?q=is%3Apr+%22Tutorial+Updates%22+is%3Aopen) that need to be merged **before ember-cli is released**. If we have missed our window then the `guides/release/tutorial/part-1/orientation.md` will show the **new ember-cli version** and when we snapshot the `release` folder (i.e. copy it to a folder with the previous release number) then the tutorial will be wrong.
```
Comment (IgnaceMaes) [inline ember-releases.md:19]: - What qualifies as `Tutorial Updates` PRs that need to be merged`
  - Any PR that is listed with that filter?
- Does this need to be checked manually, or is there a reminder/way to get pinged?

Comment (IgnaceMaes) [inline ember-releases.md:19]: - What qualifies as `Tutorial Updates` PRs that need to be merged`
  - Any PR that is listed with that filter?
- Does this need to be checked manually, or is there a reminder/way to get pinged?

Comment (MinThaMie) [reply ember-releases.md:19]: The PR that shows up when clicking on that link (and it should be one if I'm not mistaken is the one. I think getting pinged requires a lot of github  check magic to happen in our release, so I don't think that's currently worth the effort

Comment (IgnaceMaes) [reply ember-releases.md:19]: Alright, this looks good for now then

**Review by IgnaceMaes — APPROVED** [review]

**Pull request merged**